### DISCUSSION TITLE
long polling

### DISCUSSION THREAD

**Discussion by DuajDiaz** [DuajDiaz/Travel-Website#65] [Q&A] [answered]
What is the meaning of long polling in Web Development?

**Comment by Bunny77K9** [comment, marked as answer]
Long polling is a development pattern that is used to emulate a data push operation from a server to a client machine.

When long polling is operational, the client sends in a request to the server, and the data is pushed. The connection will timeout only when the data is sent to the client or after the timeout criteria are met.
